README.md

```diff
@@ -109,6 +159,79 @@ To enable/disable autostart, run
 sudo systemctl [enable|disable] bridgehead@<PROJECT>.service
 ```
 
+### Testing your new Bridgehead
+
+After starting the Bridgehead, you can watch the initialization process with the following command:
+
+```shell
+sudo journalctl -xefu bridgehead@bbmri.service
+```
+
+if this exits with the following:
+
+```
+bridgehead@bbmri.service: Main process exited, code=exited, status=1/FAILURE
+```
+
+Then you know that there was a problem with starting the Bridgehead. Scroll up the printout to find the cause of the error.
+
+Once the Bridgehead is running, you can also view the individual Docker processes with:
+
+```shell
+docker ps
+```
+
+There should be 6 Docker proceses. If there are fewer, then you know that something has gone wrong. To see what is going on, run:
+
+```shell
+docker-compose -f bbmri/docker-compose.yml logs --follow
```

Comment:
I would rather suggest to use the system units logs. For this reuse the journalctl command: `journalctl -u bridgehead@bbmri -f`
This should result in identical output